### DISCUSSION TITLE
Unable to submit code in sparksql

### DISCUSSION THREAD

**Discussion by souravbanti** [davidzajac1/zillacode#7] [General]
I am unable to submit code if I use sparksql. always get the error 'name 'spark' is not defined'

**Comment by davidzajac1** [comment]
> I am unable to submit code if I use sparksql. always get the error 'name 'spark' is not defined'

Can you post a screenshot of the code you are trying to run?

**Reply by souravbanti** [reply]
Here it is
<img width="956" alt="Screenshot 2024-09-18 001401" src="https://github.com/user-attachments/assets/71f93a69-0354-4203-ab20-c553fe1d5c48">
<img width="941" alt="image" src="https://github.com/user-attachments/assets/6fbeb21e-5ff9-4ef5-bae0-07f9498f2054">

**Comment by souravbanti** [comment]
Any resolution on this issue?

**Reply by davidzajac1** [reply]
@souravbanti looked into this one a bit. It's a bit of a tricky problem, if you want to try to get it to work it will need some code changes here: https://github.com/davidzajac1/zillacode/blob/master/spark-lambda/lambda_function.py#L39-L55
